plugins/out_s3/s3.c

```diff
@@ -2401,6 +2422,12 @@ static struct flb_config_map config_map[] = {
      "key formatters all work as expected while this feature is set to true."
     },
 
+    {
+     FLB_CONFIG_MAP_STR, "storage_class", NULL,
+     0, FLB_FALSE, 0,
+     "Specify the storage class for S3 objects."
```

Comment:
Could we have more details here? something like by default we use `S3 Standard` storage

Reply:
Thank you for your time to review this pull request!

I have created a PR for the docs with a bit more detailed explanation.
I am wondering if the same information as the documentation should be included in the help.
https://github.com/fluent/fluent-bit-docs/pull/732

Reply:
@pekeq Thanks. And yeah, we need same details here so that people can get help based on fluent bit logs.

Reply:
@zhonghui12  Thank you for your comment! I have updated the commit to include detailed help text.